lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -79,82 +80,57 @@ def expects_uri?
           term_config.term_id_expects_uri?
         end
 
-        def extract_uri_from_id(id)
-          return @uri = id if expects_uri?
+        def uri
+          return @uri if @uri.present?
+          return @uri = RDF::URI.new(id) if expects_uri?
           @uri = graph_service.subjects_for_object_value(graph: @filtered_graph, predicate: RDF::URI.new(term_config.term_results_id_predicate), object_value: id.gsub('%20', ' ')).first
         end
 
         def preds_for_term
-          { required: required_term_preds, optional: optional_term_preds }
```

Comment:
Predicate map wants to be a flat hash, so collapsed the required and optional into a single hash.